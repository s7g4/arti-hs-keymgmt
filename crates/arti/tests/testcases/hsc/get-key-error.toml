bin.name = "arti"
args = "-c ./tests/testcases/hsc/conf/hsc.toml hsc get-key --key-type=service-discovery --onion-name fpqqmiwzqiv63jczrshh4qcmlxw6gujcai3arobq23wikt7hk7ojadid.onion --generate=no --output -"
status.code = 127
