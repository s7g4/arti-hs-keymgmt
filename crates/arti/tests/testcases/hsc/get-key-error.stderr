[..]error: Service discovery key not found. Rerun with --generate=if-needed to generate a new service discovery keypair
